network/version.go

```diff
@@ -7,11 +7,16 @@ import "math"
 type Version uint
 
 const (
-	Version0 = Version(iota) // specs-actors v0.9.3
-	Version1                 // specs-actors v0.9.7
-	Version2                 // specs-actors v0.9.?
-	Version3                 // Coming soon
-	Version4                 // Who knows?
+	Version0 = Version(iota) // 00000000: genesis   (specs-actors v0.9.3)
+	Version1                 // 00041280: breeze    (specs-actors v0.9.7)
+	Version2                 // 00051000: smoke     (specs-actors v0.9.)
+	Version3                 // 00094000: ignition  (specs-actors v0.9)
+	Version4                 // 00128888: actors v2 (specs-actors v2.0.x (future))
+	Version5                 // 00148888: liftoff   (specs-actors v2.0.x (future))
```

Comment:
I don't think we should duplicate mainnet configuration like epoch numbers in here. These won't match future testnets etc.

Also I have already been burned once by describing a future version before it happened, and the description being then proven wrong.  At present, there is no intention of a version number change at liftoff.

Adding an opaque `Version5` is fine, but I'd retract all descriptions except for the specs-actors versions corresponding to actually releases that first included reference to those numbers.

Reply:
> At present, there is no intention of a version number change at liftoff.

There will be migration code (to set a network name, at least). Does this not imply a network version number? I've been assuming that's what they're for.

Reply:
> Adding an opaque Version5 is fine, but I'd retract all descriptions except for the specs-actors versions corresponding to actually releases that first included reference to those numbers.

What about the upgrade name? These versions are used for more than just specs-actors upgrades.

Reply:
I believe there has already been a migration in code that did not change the version number - the great unburning and fund redistribution. The version number only *needs* to change when actor behaviour changes.

Reply:
From what I can tell, that was the upgrade at "breeze" which bumped us from version 0 to version 1.